_episodes/10-conflicts.md

```diff
@@ -9,528 +9,407 @@ objectives:
+`update-plan`, write something new to replace both, or get rid of the change entirely. 
+Since these are both tasks we want to do, let's keep them both. Update the file so it 
+looks like this:
 
 ~~~
-Cold and dry, but everything is my favorite color
-The two moons may be a problem for Wolfman
-But the Mummy will appreciate the lack of humidity
-We added a different line in the other copy
+# about your new image collection
+
+The `cats-human-situations.csv` file contains metadata for three image objects.
+The original metadata from the source institutions has been abbreviated and made
+messier so you have something to clean up!
+
+The images are in the `images/` subdirectory of this repository.
+
+Cleanup plan:
+- Get rid of all caps in titles
+- Standardize dates
+- Standardize dimensions
+- Reconcile subjects.
+- Add access permissions
 ~~~
 {: .output}
 
-We can commit the change locally:
+To finish merging, we add save and close the file. Then add `README.md` to the changes 
+being made by the merge:
 
 ~~~
-$ git add mars.txt
-$ git commit -m "Add a line in my copy"
+git add README.md
 ~~~
 {: .bash}
 
+Let's use `git status` to see what is going on.
+
+~~~
+$ git status
 ~~~
-[master 07ebc69] Add a line in my copy
- 1 file changed, 1 insertion(+)
+{: .bash}
+~~~
+On branch master
+All conflicts fixed but you are still merging.
+  (use "git commit" to conclude merge)
+
+Changes to be committed:
+
+	modified:   README.md
 ~~~
 {: .output}
 
-but Git won't let us push it to GitHub:
+Now let's commit those changes to complete the merge.
 
 ~~~
-$ git push origin master
+git commit -m "Merge changes from update-plan branch"
 ~~~
 {: .bash}
+~~~
+[master 7bc42af] Merge changes from update-plan branch
+~~~
+{: .output}
+
+Git's ability to resolve conflicts is very useful, but conflict resolution
+costs time and effort, and can introduce errors if conflicts are not resolved
+correctly. If you find yourself resolving a lot of conflicts in a project,
+consider these technical approaches to reducing them:
+
+- Pull from upstream more frequently, especially before starting new work
+- Use topic branches to segregate work, merging to master when complete
+- Make smaller more atomic commits
+- Where logically appropriate, break large files into smaller ones so that it is
+  less likely that two authors will alter the same file simultaneously
+
+Conflicts can also be minimized with project management strategies:
+
+- Clarify who is responsible for what areas with your collaborators
+- Discuss what order tasks should be carried out in with your collaborators so
+  that tasks expected to change the same lines won't be worked on simultaneously
+- Design your workflow to minimize the potential for conflicts.
+- If the conflicts are stylistic (e.g. tabs vs. spaces), establish a
+  project convention that is governing and use code style tools (e.g.
+  `htmltidy`, `perltidy`, `rubocop`, etc.) to enforce, if necessary
```

Comment:
Was there supposed to be more to this sentence?

Reply:
that's all there was in the original

Reply:
Oh gotcha.  Then add a period and we're good to go!

Reply:
hahahaha the only one that had a period was the sentence i added (Design your workflow to minimize the potential for conflicts.). i'll add to all the others and update the pr